common/src/main/java/org/apache/drill/common/config/DrillConfig.java

```diff
@@ -138,34 +154,83 @@ public static DrillConfig create(Properties testConfigurations) {
     return create(null, testConfigurations, true);
   }
 
-  public static DrillConfig create(String overrideFileName, boolean enableServerConfigs) {
-    return create(overrideFileName, null, enableServerConfigs);
+  /**
+   * ...
+   * @param overrideFileResourcePathname
+   *          see {@link #create(String)}'s {@code overrideFileResourcePathname}
+   */
+  public static DrillConfig create(String overrideFileResourcePathname, boolean enableServerConfigs) {
+    return create(overrideFileResourcePathname, null, enableServerConfigs);
   }
 
-  private static DrillConfig create(String overrideFileName, Properties overriderProps, boolean enableServerConfigs) {
-    overrideFileName = overrideFileName == null ? CommonConstants.CONFIG_OVERRIDE : overrideFileName;
-
-    // first we load defaults.
+  /**
+   * ...
```

Comment:
>  I'm not even sure that it makes sense to document these methods individually. 

If a method is not documented individually, then viewing that method's documentation isn't going to yield much information--remember that people don't see comments just in the context of the surrounding source code, but also in IDEs (e.g., showing one method's documentation at a time) and in generated Javadoc pages.